doc/express-reboot/Cisco_8000_Express_Reboot_HLD.md

```diff
@@ -0,0 +1,123 @@
+#
+# Sonic Express reboot HLD Spec
+
+## Overview
+
+The goal of Sonic express reboot is to be able to restart and upgrade SONiC software with sub-second data plane interruption. This is achieved by not initializing external phys, and only initializing NPU after receiving all configuration updates. This document covers updates needed in Sonic and SAI interface. 
+
+## fast-reboot script
+
+The existing fast-reboot script ((https://github.com/sonic-net/sonic-utilities/blob/master/scripts/fast-reboot) that is used to trigger fast-reboot/warm-boot will be enhanced to support express-reboot. Major changes are:
+
+1.	Enforce express-reboot is only applicable to Cisco 8000 platforms at the moment.
+2.	During express-reboot shutdown path, syncd_request_shutdown will be called with a new option “-pxe” to inform syncd pre-shutdown for express boot mode. The actual calls will be “docker exec syncd /usr/bin/syncd_request_shutdown –pxe”.
```

Comment:
What does -pxe stand for? Why not use -exp as it is the prefix of express reboot?

Reply:
HI Ying, it stands for pre-expressboot. This is the notification before reboot.